measure_extinction/utils/merge_spex_spec.py

```diff
@@ -28,27 +29,35 @@
                                                 'data/Out')
     )
     parser.add_argument("--outname", help="Output filebase")
-    parser.add_argument("--png", help="save figure as a png file", action="store_true")
-    parser.add_argument("--eps", help="save figure as an eps file", action="store_true")
-    parser.add_argument("--pdf", help="save figure as a pdf file", action="store_true")
     args = parser.parse_args()
 
-    sfilename = "%s/%s.txt" % (args.inpath, args.starname)
-    stable = Table.read(
-        sfilename,
-        format="ascii",
-        names=[
-            "WAVELENGTH",
-            "FLUX",
-            "ERROR",
-            "FLAG",
-        ],
-    )
-    stable = [stable]
-    rb_stis_opt = merge_spex_obsspec(stable)
-    if args.outname:
-        outname = args.outname
+    # check which data are available
+    filename_S = "%s/%s_SXD.txt" % (args.inpath, args.starname)
+    filename_L = "%s/%s_LXD.txt" % (args.inpath, args.starname)
+    if not os.path.isfile(filename_S):
+        filenames = [filename_L]
+        if not os.path.isfile(filename_L):
+            print("No spectra could be found for this star!")
+    elif not os.path.isfile(filename_L):
+        filenames = [filename_S]
     else:
-        outname = args.starname.lower()
-    spex_file = "%s_spex.fits" % (outname)
-    rb_stis_opt.write("%s/%s" % (args.outpath, spex_file), overwrite=True)
+        filenames = [filename_S, filename_L]
+
+    for filename in filenames:
+        table = Table.read(
+            filename,
+            format="ascii",
+            names=[
+                "WAVELENGTH",
+                "FLUX",
+                "ERROR",
+                "FLAG",
+                ],
+                )
+        rb_stis_opt = merge_spex_obsspec(table)
```

Comment:
For consistency and to avoid confusion, this variable should be `rb_spex_opt`.

Reply:
> For consistency and to avoid confusion, this variable should be `rb_spex_opt`.

Indeed! I copied it and forgot to change that. In fact, I am not sure why it is called rb/opt. What does rb or opt stand for?

Reply:
Not sure.  I think it referred to "ralph bohlin" as he was the one that reduced the data.

Reply:
And he had a special format he delivered it to us in.  Hence the tag.  Not necessarily needed here, but not a problem either.  :-)